syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py

```diff
@@ -64,6 +64,24 @@ def download(target_path: Path, version: str):
         logging.info(f"File {target_file} found, skipping download.")
 
 
+_FCNET_SEED_KEY = "replication"
+
+
+def _check_whether_fcnet(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id == "fcnet"
+
+
+def _check_whether_iaml(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id.startswith("iaml_")
+
+
+def _check_whether_rbv2(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id.startswith("rbv2_")
+
+
+# TODO: For fcnet, we also need to determine the "elapsed_time" metric from
+# "runtime", the latter is the time for all 100 epochs. Here, we assume that
+# "elapsed_time" for r epochs is `r / 100` times "runtime".
```

Comment:
Regarding FCNET, we could also not expose it given that YAHPO fits a surrogate on a grid of evaluations which is unlikely to provide good results (and was not mentioned in the paper). 
Given this, I would have a preference to leave it out as it is one less thing to maintain but I leave the decision up to you.

Reply:
This will simplify the code